docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
+
+Functional Groups in *stk*
+--------------------------
+
+https://www.youtube.com/watch?v=IaLIN7jXyKQ&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=2
+
+Cage Building
+-------------
+
+https://www.youtube.com/watch?v=sc1Kf6vzTSA&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=3
+
+
+Hierarchical Construction with *stk*
+------------------------------------
```

Comment:
This video expands on the use hierarchical construction in _stk_. This includes an implementation of the example from the most recent publication and the construction a rotaxane/cage hybrid!

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .